.buildkite/build_benchmark.sh

```diff
@@ -13,6 +14,23 @@ if [[ "${JULIAHUBREGISTRY_BENCHMARK_TARGETS[*]}" =~ "${1}" ]]; then
 	julia -e 'using Pkg; Pkg.Registry.add(); Pkg.Registry.status()'
 fi
 
+if [[ "${OPENMODELICA_BENCHMARK_TARGETS[*]}" =~ "${1}" ]]; then
+	echo "--- :toolbox: Installing OpenModelica"
+	adduser --force-badname --system --home /nonexistent  \
+            --no-create-home --quiet _apt
+	apt-get update
+	apt-get install --yes ca-certificates curl gnupg
+	curl -fsSL http://build.openmodelica.org/apt/openmodelica.asc | gpg --dearmor -o /usr/share/keyrings/openmodelica-keyring.gpg
+
+	echo "deb [arch=arm64 signed-by=/usr/share/keyrings/openmodelica-keyring.gpg] https://build.openmodelica.org/apt \
```

Comment:
```suggestion
	echo "deb [arch=amd64 signed-by=/usr/share/keyrings/openmodelica-keyring.gpg] https://build.openmodelica.org/apt \
```

We were downloading the package listings for the wrong architecture.  :P

Reply:
Good catch!